app/views/slack/ticket_create/fr.md.erb

```diff
@@ -0,0 +1,9 @@
+# #{ticket.title}
+_<#{config.http_type}://#{config.fqdn}/#ticket/zoom/#{ticket.id}|Ticket##{ticket.number}>: Créé par #{current_user.longname} at #{ticket.updated_at}_
```

Comment:
The `at` looks like a typo, could this be? Can you check this and the rest again, please?